Target-MVD-Mathias/Extensions/UIColorExtension.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIColorExtension.swift
+//  Target-MVD-Mathias
+//
+//  Created by Mathias Cabano on 8/10/17.
+//  Copyright © 2017 TopTier labs. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+    
+    class func createColor(red: CGFloat, green: CGFloat, blue: CGFloat) -> UIColor {
```

Comment:
I think this method could be more extensible if you add the alpha with a default param of 1.